src/containers/AttributionPanel.js

```diff
@@ -46,7 +46,7 @@ const enhance = compose(
   withStyles(styles),
   withTranslation(),
   connect(mapStateToProps),
-  withPlugins('ManifestInfo'),
+  withPlugins('AttributionPanel'),
```

Comment:
Nice catch

Reply:
It actually caused a bug in our instance, since we had a plugin targetting `ManifestInfo` that suddenly broke the attribution panel :sweat_smile: